zerver/openapi/zulip.yaml

```diff
@@ -519,16 +519,19 @@ paths:
                                     - type: object
                                       additionalProperties: false
                                       description: |
-                                        When the delivery email of a user changes.
+                                        When the delivery email of a user changes or `email_address_visibility`
+                                        setting of a user changes.
 
-                                        Note: This event is only visible to admins.
+                                        **Changes**: Prior to Zulip 7.0 (feature level 160), this event was sent
+                                        to the target user only and only when delivery email of the user was changed.
```

Comment:
"was sent only to organization administrators and the affected user, and this
event would only be triggered by changing the affected user's delivery email."

Reply:
This was actually sent to only affected user as per the [code here](https://github.com/zulip/zulip/blob/a6abf959bb68bcc6275ce94601a1d4ab21234319/zerver/actions/user_settings.py#L69) and the documentation was incorrect.

Also mentioned about `None` case.